services/base.go

```diff
@@ -92,3 +95,14 @@ func reportFailed(failed []string, description string) {
 		fmt.Println(color.RedString(strings.Join(failed, "\n")))
 	}
 }
+
+// InitBaseURL allows updating the base URL from outside the package
+func InitBaseURL(sourceURL, targetURL string) {
```

Comment:
This func is not required, when the source and target URL is empty we must set the BaseURL to it.